File: server/iih_pub/iih.ci_pub/src/main/java/iih/ci/mr/nurinfodto/d/NurInfoDTO.java
package iih.ci.mr.nurinfodto.d;

import xap.mw.core.data.*;
import xap.mw.coreitf.d.*;
import java.math.BigDecimal;

/**
 * 护士信息DTO DTO数据 
 * 
 */
public class NurInfoDTO extends BaseDTO {
	private static final long serialVersionUID = 1L;
	
	/**
	 * 护士编码
	 * @return String
	 */
	public String getCode_nur() {
		return ((String) getAttrVal("Code_nur"));
	}
	/**
	 * 护士编码
	 * @param Code_nur
	 */
	public void setCode_nur(String Code_nur) {
		setAttrVal("Code_nur", Code_nur);
	}
	/**
	 * 护士名称
	 * @return String
	 */
	public String getNur() {
		return ((String) getAttrVal("Nur"));
	}
	/**
	 * 护士名称
	 * @param Nur
	 */
	public void setNur(String Nur) {
		setAttrVal("Nur", Nur);
	}
}

File: client/iih_stub/iih.ci_stub/iih.ci.mr/nurinfodto/d/NurInfoDTO.cs
using System;
using xap.mw.core.data;
using xap.mw.coreitf.d;

namespace iih.ci.mr.nurinfodto.d
{
    /// <summary>
    /// NurInfoDTO 的摘要说明。
    /// </summary>
    public class NurInfoDTO : BaseDTO {

        public NurInfoDTO() {
 
        }

        /// <summary>
        /// 护士编码
        /// </summary>
		public string Code_nur {
            get { return getAttrVal<string>("Code_nur",null); }
            set { setAttrVal<string>("Code_nur", value); }
        }

        /// <summary>
        /// 护士名称
        /// </summary>
		public string Nur {
            get { return getAttrVal<string>("Nur",null); }
            set { setAttrVal<string>("Nur", value); }
        }
        /// <summary>
        /// 返回属性列表
        /// </summary>
        /// <returns></returns>
        public override string[] getAttrNames()
        {
            return new string[]{ "Code_nur", "Nur"};
        }
        
        /// <summary>
        /// 返回全路径DO类名
        /// </summary>
        /// <returns></returns>
        public override string getFullClassName()
        {
            return "iih.ci.mr.nurinfodto.d.NurInfoDTO";
        }
    }
}
